x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtTestCase.java

```diff
@@ -587,4 +585,16 @@ public ThreadContext createThreadContext(final CharSequence jwt, final CharSeque
         }
         return requestThreadContext;
     }
+
+    static Path resolvePath(final String relativePath) {
+        try {
+            final URL url = JwtTestCase.class.getResource(relativePath);
+            if (url != null) {
+                return PathUtils.get(url.toURI()).toAbsolutePath().normalize();
+            }
+        } catch (Exception e) {
+            LOGGER.error("resource not found: " + relativePath, e);
+        }
```

Comment:
I think we should throw error in this case. There is no good reason to proceed further if exception happens.